tests/integration_test.py

```diff
@@ -1073,6 +1082,64 @@ def runTest(self):
         self.assertTrue(sock.fileno() > -1)
 
 
+class TestRunContainerReadingSocket(BaseTestCase):
```

Comment:
yeah, that comes about because next is a special word in python, so I write nxt to not overwrite it.